### DISCUSSION TITLE
可以部署到自己搭建的内网服务器上吗？不使用阿里云和腾讯云

### DISCUSSION THREAD

**Discussion by IcyVizi** [midwayjs/midway#3369] [Q&A] [answered]
### Describe the problem（描述问题）

需求是开发内网应用，无法访问公有网络，所以想问下有没有部署到内网的相关文档。
比如，是否支持部署到 OpenWhisk 平台？

### Midway Versions（Midway 版本）

latest

**Comment by czy88840616** [comment]
如果是标准应用，https://midwayjs.org/docs/deployment 这里写的很清楚了，不管是使用 docker 还是普通的容器，或者其他云环境。如果是 Serverless 应用，目前仅支持阿里云。

**Reply by IcyVizi** [reply, marked as answer]
谢谢，找到了，一体化项目就是我想找的。
- https://midwayjs.org/docs/quickstart
- https://midwayjs.org/docs/hooks/intro
